boilerplates/be-node-express/Jenkinsfile

```diff
@@ -9,18 +9,18 @@ node {
   dockerRegistry = env.DOCKER_REGISTRY
 }
 
-library identifier: 'ods-library@production', retriever: modernSCM(
+library identifier: 'ods-library@feature/tailor-deploy-debug', retriever: modernSCM(
```

Comment:
@henrjk I would recommend the identifier to match this PR source branch name, or to point to production. Anyway, as you already mentioned, if the purpose of this PR is only for researching and proposing yet it is fine for me!

Reply:
Yes, I wanted to remove this change (so that it would be production) once the fix (or something equivalent) in branch feature/tailor-deploy-debug is merged.